Sources/SpeziAccessGuard/CodeViews/CodeOptions.swift

```diff
@@ -95,18 +88,70 @@ public struct CodeOptions: OptionSet, Codable, CaseIterable, Identifiable {
     public init(rawValue: Int) {
         self.rawValue = rawValue
     }
+
+    /// Verifies that the code is only digits for numeric code types
+    private func verifyOnlyDigits(ofCode code: String) -> Bool {
```

Comment:
Wondering if we should generalize this function a bit as it is called beyond just verifying for digits and returns true if a code also contains normal characters despite its name.
```suggestion
    private func verifyAllowedCharacters(ofCode code: String) -> Bool {
```